.vscode/settings.json

```diff
@@ -0,0 +1,116 @@
+{
+	// VSCode Custom Settings
+	"typescript.tsdk": "node_modules/typescript/lib",
+	// ESLint
+	"eslint.format.enable": true,
+	"eslint.validate": [
+		"javascript",
+		"typescript",
+		"javascriptreact",
+		"typescriptreact"
+	],
+	"[javascript]": {
+		"editor.defaultFormatter": "dbaeumer.vscode-eslint",
+	},
+	"[typescript]": {
+		"editor.defaultFormatter": "dbaeumer.vscode-eslint",
+	},
+	"[javascriptreact]": {
+		"editor.defaultFormatter": "dbaeumer.vscode-eslint",
+	},
+	"[typescriptreact]": {
+		"editor.defaultFormatter": "dbaeumer.vscode-eslint",
+	},
+	// PHP Code Sniffer
+	"phpCodeSniffer.autoExecutable": true,
+	"phpCodeSniffer.standard": "Custom",
+	"phpCodeSniffer.standardCustom": "WordPress-VIP-Go,WordPress-Docs",
+	// PHP Code Beautifier
+	"phpcbf.standard": "WordPress-VIP-Go,WordPress-Docs",
+	"[php]": {
+		"editor.defaultFormatter": "obliviousharmony.vscode-php-codesniffer"
+	},
+	// PHPStan
+	"phpstan.timeout": 30000,
+	"phpstan.ignoreErrors": [
+		".*The Xdebug PHP extension is active.*", ".*slow down performance.*", ".*breakpoints.*",
```

Comment:
If `Xdebug` is setup in local environment then PHPStan raises this error and stops the execution so we have added this error in ignore list so that PHPStan will complete its analyzation no matter what is the state of Xdebug.